.github/workflows/build.yml

```diff
@@ -252,7 +252,14 @@ jobs:
         # On Mac/ARM, the JSON output disagrees in the 16th and 17th digit
         #  with the JSON output on Linux/Intel and Mac/Intel.
         if [ ${{ matrix.arch }} = "mac-arm64" ] ; then
-          find tests -name '*.json' | xargs rm
+           find tests -name '*.json' | xargs rm
+        fi
+
+        # Another ugly hack: remove the *.errout file generated for the CBDSP
+        # test on Linux. Apparently, NaN log likelihoods are unsigned on macOS
```

Comment:
As far as I know, all platforms allow both positive and negative NaNs.  Are the unsigned NaNs actually positive NaNs?

Is there some known feature of Mac/Windows and not Linux that results in a different sign here?  Or do we just observe different signs in practice?

Reply:
We just observe different signs in practice. This line from the standard output of the CBDSP test when run on macOS oe (emulated) Windows

```
Could not compute lnProb for node 'timetree': lnProb = nan
```

reads

```
Could not compute lnProb for node 'timetree': lnProb = -nan
```

on Linux.